### ISSUE TITLE
Fatal Exception: Package android does not belong to 10251

### ISSUE THREAD

**Issue by RodolfoGS** [flurry/react-native-flurry-sdk#33]
**Describe the bug**
I have this crash on Crashlytics (Android), I don't know how to reproduce it.

```
Fatal Exception: java.lang.SecurityException: Package android does not belong to 10251
       at android.os.Parcel.createExceptionOrNull(Parcel.java:2373)
       at android.os.Parcel.createException(Parcel.java:2357)
       at android.os.Parcel.readException(Parcel.java:2340)
       at android.os.Parcel.readException(Parcel.java:2282)
       at android.net.IConnectivityManager$Stub$Proxy.getNetworkCapabilities(IConnectivityManager.java:2456)
       at android.net.ConnectivityManager.getNetworkCapabilities(ConnectivityManager.java:1385)
       at com.flurry.sdk.av.getNetworkTypeM(SourceFile:518)
       at com.flurry.sdk.av.b(SourceFile:191)
       at com.flurry.sdk.av.c(SourceFile:418)
       at com.flurry.sdk.av.<init>(SourceFile:163)
       at com.flurry.sdk.n.<init>(SourceFile:1071)
       at com.flurry.sdk.n.a(SourceFile:48)
       at com.flurry.sdk.bn.<init>(SourceFile:43)
       at com.flurry.sdk.bs.<init>(SourceFile:24)
       at com.flurry.sdk.ey.<init>(SourceFile:38)
       at com.flurry.sdk.fb.<init>(SourceFile:1014)
       at com.flurry.sdk.fb.a(SourceFile:31)
       at com.flurry.android.FlurryAgent$Builder.build(SourceFile:2120)
       at com.flurry.android.reactnative.FlurryModule.build(FlurryModule.java:114)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
       at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:151)
       at com.facebook.react.bridge.queue.NativeRunnable.run(NativeRunnable.java)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
       at android.os.Looper.loop(Looper.java:223)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:226)
       at java.lang.Thread.run(Thread.java:923)
```

**This Bug is About**
Please choose the closest item by replacing `[ ]` with `[x]`.
- [ ] Integration
- [x] Analytics
- [ ] Messaging/Push
- [ ] Remote Configuration
- [ ] Flurry Website/Dashboard
- [ ] Other

**Platform**
Please choose the platform(s) that you are having the issue by replacing `[ ]` with `[x]`.
- [x] Android
- [ ] iOS
- [ ] tvOS

**Environment**
Please tell us the versions of SDKs you are using. If you are not sure about React Native version, you can run `react-native --version` under your project.
* React Native version: 0.63.4
* react-native-flurry-sdk version: 6.2.0

**To Reproduce**
I have this crash on Crashlytics (Android), I don't know how to reproduce it. Sorry.

**Screenshots**
From Crashlytics

**Additional context**
Are you using Flurry Push/Messaging? No
(iOS) Are you using CocoaPods for React Native integration? Yes

**Stacktrace**
```
Fatal Exception: java.lang.SecurityException: Package android does not belong to 10251
       at android.os.Parcel.createExceptionOrNull(Parcel.java:2373)
       at android.os.Parcel.createException(Parcel.java:2357)
       at android.os.Parcel.readException(Parcel.java:2340)
       at android.os.Parcel.readException(Parcel.java:2282)
       at android.net.IConnectivityManager$Stub$Proxy.getNetworkCapabilities(IConnectivityManager.java:2456)
       at android.net.ConnectivityManager.getNetworkCapabilities(ConnectivityManager.java:1385)
       at com.flurry.sdk.av.getNetworkTypeM(SourceFile:518)
       at com.flurry.sdk.av.b(SourceFile:191)
       at com.flurry.sdk.av.c(SourceFile:418)
       at com.flurry.sdk.av.<init>(SourceFile:163)
       at com.flurry.sdk.n.<init>(SourceFile:1071)
       at com.flurry.sdk.n.a(SourceFile:48)
       at com.flurry.sdk.bn.<init>(SourceFile:43)
       at com.flurry.sdk.bs.<init>(SourceFile:24)
       at com.flurry.sdk.ey.<init>(SourceFile:38)
       at com.flurry.sdk.fb.<init>(SourceFile:1014)
       at com.flurry.sdk.fb.a(SourceFile:31)
       at com.flurry.android.FlurryAgent$Builder.build(SourceFile:2120)
       at com.flurry.android.reactnative.FlurryModule.build(FlurryModule.java:114)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
       at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:151)
       at com.facebook.react.bridge.queue.NativeRunnable.run(NativeRunnable.java)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
       at android.os.Looper.loop(Looper.java:223)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:226)
       at java.lang.Thread.run(Thread.java:923)

Caused by android.os.RemoteException: Remote stack trace:
	at android.app.AppOpsManager.checkPackage(AppOpsManager.java:7698)
	at com.android.server.ConnectivityService.getNetworkCapabilities(ConnectivityService.java:1668)
	at android.net.IConnectivityManager$Stub.onTransact(IConnectivityManager.java:978)
	at android.os.Binder.execTransactInternal(Binder.java:1154)
	at android.os.Binder.execTransact(Binder.java:1123)
```

**Device**
* Brand: Google
* Model: Pixel 3 XL
* Android Version: 11

**Comment by poting-oath** [comment]
@RodolfoGS Thanks for the report! Yes, this is a known issue for Android 11. The fix will be available in the next release.

Do you know when it happened, is it under Wifi, Cell, or no network? Does your app have `android.permission.ACCESS_NETWORK_STATE`?

**Comment by RodolfoGS** [comment]
I don't know how to reproduce, I see those crashes on Firebase. And I don't know if the device is under Wifi or Cell, Firebase don't give me that info, sorry :(

But I could give you the report from another device that was recently crash.
- Brand: Samsung
- Model: Galaxy S10+
- Android Version: 11

And yes, I have the `android.permission.ACCESS_NETWORK_STATE` permission.

**Comment by poting-oath** [comment]
@RodolfoGS Fix has been released in version 6.3.0. Thanks!
